docs/getting_started/js_utils.md

```diff
@@ -0,0 +1,151 @@
+// Note: It will ask the user for permission, and will error out if this is not given.
+camera.start();
+```
+### Control Panel Utilities
+To showcase and monitor the model in elegant ways, MediaPipe has this Control Panel module.
+This panel is also convenient to use and set up.
+```js
+// Pass in a <div>
+new ControlPanel(controlsElement, {
+      // Whether to invert the camera, defaults to false
+      selfieMode: true,
+      // Maximum Number of Hands to detect, a placeholder, only when using Hands API
+      maxNumHands: 2,
+      // Minimum Confidence score
+      minDetectionConfidence: 0.5,
+      // Minimum Tracking score
+      minTrackingConfidence: 0.5
+    })
+    .add([
+      // a StaticText is simply a label
+      new StaticText({title: 'MediaPipe Hands'}),
+      // a Toggle is one with only 2 options (i.e., true or false)
+      new Toggle({title: 'Selfie Mode', field: 'selfieMode'}),
+      // A slider can have multiple options. This will have the options 1,2,3 and 4. 
+      // Range determines maximum and minimum allowed value
+      // Step determines the difference between two options
+      new Slider(
+          {title: 'Max Number of Hands', field: 'maxNumHands', range: [1, 4], step: 1}),
+      new Slider({
+        title: 'Min Detection Confidence',
+        field: 'minDetectionConfidence',
+        range: [0, 1],
+        step: 0.01
+      }),
+      new Slider({
+        title: 'Min Tracking Confidence',
+        field: 'minTrackingConfidence',
+        range: [0, 1],
+        step: 0.01
+      }),
+    ])
+    // This is run when an option is updated. 
+    // the options object will contain parameters for all the above in a array.
+    .on(options => {
+      videoElement.classList.toggle('selfie', options.selfieMode);
+      hands.setOptions(options);
+    });
+
+```
+
+[Ho-pg]: ../solutions/holistic#javascript-solution-api
+[F-pg]: ../solutions/face_mesh#javascript-solution-api
+[H-pg]: ../solutions/hands#javascript-solution-api
+[P-pg]: ../solutions/pose#javascript-solution-api
+[Ho-npm]: https://www.npmjs.com/package/@mediapipe/holistic
+[F-npm]: https://www.npmjs.com/package/@mediapipe/face_mesh
+[H-npm]: https://www.npmjs.com/package/@mediapipe/hands
+[P-npm]: https://www.npmjs.com/package/@mediapipe/pose
+[draw-npm]: https://www.npmjs.com/package/@mediapipe/pose
+[cam-npm]: https://www.npmjs.com/package/@mediapipe/pose
+[ctrl-npm]: https://www.npmjs.com/package/@mediapipe/pose
+[Ho-jsd]: https://www.jsdelivr.com/package/npm/@mediapipe/holistic
+[F-jsd]: https://www.jsdelivr.com/package/npm/@mediapipe/face_mesh
+[H-jsd]: https://www.jsdelivr.com/package/npm/@mediapipe/hands
+[P-jsd]: https://www.jsdelivr.com/package/npm/@mediapipe/pose
+[Ho-pen]: https://code.mediapipe.dev/codepen/holistic
+[F-pen]: https://code.mediapipe.dev/codepen/face_mesh
+[H-pen]: https://code.mediapipe.dev/codepen/hands
+[P-pen]: https://code.mediapipe.dev/codepen/pose
+[Ho-demo]: https://mediapipe.dev/demo/holistic
```

Comment:
is it worth including a link to the holistic remote demo, too? (https://mediapipe.dev/demo/holistic_remote/)
Frustratingly, I can't find source code around for it, but shows additional applications using the holistic API 

(BTW, if anyone from the mediapipe team sees this and doesn't mind uploading it to code pen, I, and presumably others, would be forever indebted!)

Reply:
Also! Thanks for pulling this together @djthegre8!

Reply:
@afogel sure...

Reply:
Remote demo doesn't have a npm or a codepen, as you mentioned (I think they have a private npm, they asked me to log in when I put the link), so I could put it up, but not in the table, just as a see also I guess